tests/Oryx.BuildImage.Tests/Php/PhpSampleAppsTest.cs

```diff
@@ -62,7 +62,6 @@ public void GeneratesScript_AndBuilds_TwigExample_InLtsVersionsBuildImage()
         [InlineData(PhpVersions.Php74Version)]
         [InlineData(PhpVersions.Php73Version)]
         [InlineData(PhpVersions.Php72Version)]
-        [InlineData(PhpVersions.Php70Version)]
```

Comment:
why did we remove this?

Reply:
Not supporting php|7.3 at all.. there are very minimal number of sites.